src/Main.jsx

```diff
@@ -27,21 +27,37 @@ export default class Main extends Component {
 
   componentDidMount() {
     const { plugin } = this.props;
+    const urlParts = [];
 
     const {
       itemType,
       fields,
       locale,
       field,
       parameters: {
-        global: { developmentMode },
+        global: { developmentMode, useLocalePath, skipDefaultLocalePath },
       },
     } = plugin;
 
+    if (useLocalePath && !(skipDefaultLocalePath && plugin.site.attributes.locales[0] === locale)) {
+      urlParts.push(locale);
+    }
+
     const slugField = itemType.relationships.fields.data
       .map(link => fields[link.id])
       .find(f => f.attributes.field_type === 'slug');
 
+    const modelslugField = itemType.relationships.fields.data
```

Comment:
I'm not really sure about the name `modelslug`. What do you think about one of `frontendPath`, `frontendSlug`, `sitePath` or `siteSlug`?

Reply:
I think it should specifically mention 'model',  to highlight that it concerns the translated model in the url and not the actual item. So maybe 'modelPath' ?

Reply:
I disagree as the model is part of the CMS. The value you're setting only has something to do with the frontend. Obviously this value has to be set in a field which belongs to a model, but a field can only exist on a model so no need to explicitly state that. It's not the path of the model, it's (a part) of the path of the frontend page.

Just to be sure, my comment is about the `api_key`, so actually about [Line 52](https://github.com/greenberrynl/datocms-plugin-gatsby-cloud/pull/1/files/eea3541ed9221569f05783a9b8f639cb79f26a32#diff-a79f797206d5df2a028cd51eb8716f62f19fc6dd9efd319a634ba95dd826ed8fR52)

Reply:
I like `frontendSlug`